java/code/src/com/suse/manager/maintenance/MaintenanceManager.java

```diff
@@ -246,6 +246,34 @@ protected void save(MaintenanceCalendar calendar) {
             .list();
     }
 
+    /**
+     * List Maintenance Schedules belonging to the given User
+     * @param user the user
+     * @return a list of Maintenance Schedules
+     */
+    @SuppressWarnings("unchecked")
```

Comment:
Is this annotation needed?

Reply:
All the other queries in this file added by mc also seem to have it. And it gets rid of annoying warnings

Reply:
What warnings? I don't see any.

Reply:
It displays a `unchecked assignment` warning if removed.
Anyway I am not opposed to removing it, but we should stay consistent.
Either remove or add it everywhere.